resources/services/ec2/byoip_cidrs.go

```diff
@@ -68,14 +68,8 @@ func fetchEc2ByoipCidrs(ctx context.Context, meta schema.ClientMeta, parent *sch
 	c := meta.(*client.Client)
 	// DescribeByoipCidrs does not work in next regions, so we ignore them.
 	if _, ok := map[string]struct{}{
-		"ap-northeast-2": {},
-		"af-south-1":     {},
 		"cn-north-1":     {},
 		"cn-northwest-1": {},
```

Comment:
I wonder if there is a lesson we can draw from this for the future: maybe we should still make the call, but ignore the specific error that gets returned from unsupported regions? I don't know if that would have been possible though, necessarily.

Reply:
This was definitely an early way that we handled regional differences. I agree, today we usually would make the call and catch the specific error that would indicate the feature is not supported in the region. That being said the China and us-gov partitions we are not able to test in as we cannot get accounts in those partitions